v-next/hardhat/src/internal/builtin-plugins/solidity-test/formatters.ts

```diff
@@ -116,9 +116,11 @@ function formatTrace(trace: CallTrace): NestedArray<string> {
   let closingLine: string | undefined;
   if (kind === CallKind.Create) {
     openingLine = `[${gasUsed}] ${chalk.yellow("→ new")} ${contract}`;
-    if (formattedInputs !== undefined) {
-      openingLine = `${openingLine}@${formattedInputs}`;
-    }
+    // TODO: Uncomment this when the formattedInputs starts containing
+    // the address of where the contract was deployed instead of the code.
+    // if (formattedInputs !== undefined) {
+    //   openingLine = `${openingLine}@${formattedInputs}`;
+    // }
```

Comment:
> Contract deployments is showing something like new Counter@0x6080604052348015600e575f5ffd5b506103cf8061001c5f395ff3fe6080604..., where the full deployment code is included after the @. Instead, this should show the address where Counter was deployed. I don't know if the problem here is in the traces returned by EDR or in Hardhat's formatter.

I addressed this suggestion here. AFAICT, we don't have access to an address of the deployed contract right now.